ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization.jsx

```diff
@@ -0,0 +1,128 @@
+    userStore.getOrganizationUsers();
+  }
+
+  addRegisteredUser = () => {
+    const { userEmail } = this.state;
+    if (isEmail(userEmail)) {
+      const { stores } = this.props;
+      const { userStore } = stores;
+      userStore.addUserToOrganization(userEmail);
+    }
+  };
+
+  userEmailChangeCallback = (value) => {
+    this.setState({ userEmail: value, userEmailValid: isEmail(value) });
+  };
+
+  render() {
+    const { userEmail, userEmailValid } = this.state;
+    const { stores } = this.props;
+    const { userStore } = stores;
+    const namespace = userStore.userOrganizationNamespace;
+    const organizations = userStore.userOrganizations;
+    const organizationUsers = userStore.userOrganizationUsers;
+    return (
+      <div className="profile-container" id="profile-organization">
+        <span>
+          <div className="description">
+            Allow existing users to access the elements in the selected organization.
+          </div>
+          <div className="section-header">
+            <div className="column name-header">Organization Name</div>
+            <div className="column id-header">Organization ID</div>
+          </div>
+          {organizations.map(item => (
+            <div className="organization-info" key={`organization-info-${item.namespace}`}>
+              <div className={`column name ${item.namespace === namespace ? 'selected' : ''}`} id="organization-name">
+                <div
+                  className={`dot ${item.namespace !== namespace ? 'disabled' : 'selected'}`}
+                  id={`organization-checkbox-${item.namespace.toLowerCase()}`}
+                />
+                {item.name}
+              </div>
+              <div className="column id" id="organization-id">
+                {item.namespace}
+              </div>
+            </div>
+          ))}
+          <span>
+            <div className="section-header adding-user">
+              <div className="column name-header">Add registered user</div>
+            </div>
+            <div className="description">
+              People you add here must have an OTA Connect account. If you are unsure check with that person first.
+            </div>
+            <div className="adding-user-content">
+              <Form className="adding-user-form" id="add-registered-user-form">
+                <SearchBar
+                  additionalClassName="white dark-border"
+                  changeAction={this.userEmailChangeCallback}
+                  id="add-registered-user-search-bar"
+                  placeholder={'Add by email address'}
+                  value={userEmail}
+                />
+              </Form>
+              <Button
+                htmlType="button"
+                className="ant-btn ant-btn-add-registered-user"
+                disabled={!userEmailValid}
+                id="button-add-registered-user"
+                onClick={(event) => {
+                  event.preventDefault();
```

Comment:
Why not just move it to the called function, to keep the `render()` method cleaner?

Reply:
Done